CMakeLists.txt

```diff
@@ -0,0 +1,376 @@
+cmake_minimum_required(VERSION 3.20)
+
+# Project definition
+project(SuperNOVAS
+    VERSION 1.5.0
+    DESCRIPTION "SuperNOVAS astrometry library"
+    HOMEPAGE_URL "https://smithsonian.github.io/SuperNOVAS/"
+    LANGUAGES C
+)
+
+# Set C standard
+set(CMAKE_C_STANDARD 99)
```

Comment:
I'd comment that. I use the C standard only to check that we can compile for c99, but there is no reason to not compile with the compiler default language standard.